sunpy/instr/goes.py

```diff
@@ -122,15 +137,15 @@ def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH
 
     Returns
     -------
-    goeslc.data.temperature : pandas.core.series.Series
-        Array of temperature values [MK]
-
-    goeslc.data.em : pandas.core.series.Series
-        Array of volume emission measure values [10**49 cm**-3]
+    lc_new : `sunpy.lightcurve.LightCurve`
+        Contains same metadata and data as input GOESLightCurve with the
+        following two additional data columns:
```

Comment:
From my experimentation, you could infact put these code snippets in the verbatim backticks. Hopefully that will not cause sphinx to explode. If it does, we can change them to italics.